ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml

```diff
@@ -8,6 +8,7 @@ ref:
       memory: 600Mi
     limits:
       memory: 2Gi
+  termination_grace_period_seconds: 30
```

Comment:
I think this is the default value, so it could be safely omitted (as for the other steps)

Reply:
I could not find the place it is set by default. 

It seems omittable;
https://github.com/openshift/ci-tools/blob/b1df36ba835057ca830f1b835fb3f849873ea39f/pkg/api/types.go#L628

@stevekuznetsov suggested setting to some value.

Reply:
I saw it;

"Note that the step declares a 120s termination grace period, which is much larger than the default of 30s"

Reply:
Ok then it's safe to be removed